XCTestBootstrap/Configuration/FBTestLaunchConfiguration.m

```diff
@@ -27,14 +27,16 @@ - (instancetype)initWithTestBundlePath:(NSString *)testBundlePath applicationLau
   _testHostPath = testHostPath;
   _timeout = timeout;
   _shouldInitializeUITesting = initializeUITesting;
+  _testsToRun = testsToRun;
+  _testsToSkip = testsToSkip;
 
   return self;
 }
 
 + (instancetype)configurationWithTestBundlePath:(NSString *)testBundlePath
 {
   NSParameterAssert(testBundlePath);
-  return [[FBTestLaunchConfiguration alloc] initWithTestBundlePath:testBundlePath applicationLaunchConfiguration:nil testHostPath:nil timeout:0 initializeUITesting:NO];
+  return [[FBTestLaunchConfiguration alloc] initWithTestBundlePath:testBundlePath applicationLaunchConfiguration:nil testHostPath:nil timeout:0 initializeUITesting:NO testsToRun:nil testsToSkip:nil];
```

Comment:
This is erroneous as the expectation is that these values are never null. It would make more sense for `testsToSkip` to at least default to the empty set.